--- isdn4k-utils/vbox/acinclude.m4.orig	2002-07-06 02:11:19.000000000 +0200
+++ isdn4k-utils/vbox/acinclude.m4	2013-01-26 18:52:37.758313064 +0100
@@ -36,18 +36,18 @@
 
       if (test "${gnd_use_tcl_lib}" = "")
       then
-         gnd_1st_tcl_lib_test="tcl8.4"
-         gnd_2nd_tcl_lib_test="tcl8.3"
+         gnd_1st_tcl_lib_test="tcl8.5"
+         gnd_2st_tcl_lib_test="tcl8.4"
          gnd_3rd_tcl_lib_test="tcl8.0"
       else
          gnd_1st_tcl_lib_test="${gnd_use_tcl_lib}"
-         gnd_2nd_tcl_lib_test="tcl8.4"
-         gnd_3rd_tcl_lib_test="tcl8.3"
+         gnd_2nd_tcl_lib_test="tcl8.5"
+         gnd_3nd_tcl_lib_test="tcl8.4"
       fi
 
       AC_CHECK_LIB(m,
          cos,
-         AC_CHECK_LIB(dl,
+         [AC_CHECK_LIB(dl,
             dlerror,
             [AC_CHECK_LIB(${gnd_1st_tcl_lib_test},
                Tcl_CreateInterp,
