--- isdn4k-utils/capifax/configure.in.orig	2002-07-04 12:25:21.000000000 +0200
+++ isdn4k-utils/capifax/configure.in	2013-01-25 22:11:27.429865260 +0100
@@ -37,5 +37,5 @@
 AC_SUBST(INSTALL)
 AC_SUBST(CONFIG_SBINDIR)
 AC_SUBST(CONFIG_MANDIR)
-AM_CONFIG_HEADER(config.h)
+AC_CONFIG_HEADERS(config.h)
 AC_OUTPUT(Makefile)
--- isdn4k-utils/capiinfo/configure.in.orig	2002-07-19 23:23:51.000000000 +0200
+++ isdn4k-utils/capiinfo/configure.in	2013-01-25 22:17:52.826523897 +0100
@@ -42,5 +42,5 @@
 AC_SUBST(I4LVERSION)
 AC_SUBST(CONFIG_SBINDIR)
 AC_SUBST(CONFIG_MANDIR)
-AM_CONFIG_HEADER(config.h)
+AC_CONFIG_HEADERS(config.h)
 AC_OUTPUT(Makefile)
--- isdn4k-utils/capiinit/configure.in.orig	2013-01-27 10:27:38.030477094 +0100
+++ isdn4k-utils/capiinit/configure.in	2013-01-27 10:29:47.987141053 +0100
@@ -24,5 +24,5 @@
 AC_SUBST(CONFIG_SBINDIR)
 AC_SUBST(CONFIG_MANDIR)
 AC_SUBST(CONFIG_FIRMWAREDIR)
-AM_CONFIG_HEADER(config.h)
+AC_CONFIG_HEADERS(config.h)
 AC_OUTPUT(Makefile)
--- isdn4k-utils/vbox/configure.in.orig	2013-01-27 10:27:38.033810427 +0100
+++ isdn4k-utils/vbox/configure.in	2013-01-27 10:30:13.897140511 +0100
@@ -7,7 +7,7 @@
 dnl #------------------------------------------------------------------------#
 AC_INIT(vbox,[2.0.0BETA5])
 AM_INIT_AUTOMAKE
-AM_CONFIG_HEADER(src/config.h)
+AC_CONFIG_HEADERS(src/config.h)
 AC_CONFIG_SRCDIR(src/vboxgetty.h)
 
 VERDATE="17-NOV-98"
