diff -Naur isdn4k-utils.orig/pppdcapiplugin/Makefile isdn4k-utils/pppdcapiplugin/Makefile
--- isdn4k-utils.orig/pppdcapiplugin/Makefile	Wed Jul  3 02:00:28 2002
+++ isdn4k-utils/pppdcapiplugin/Makefile	Mon Oct  7 12:18:25 2002
@@ -12,7 +12,7 @@
 export CAPIINC=$(TOPDIR)/../capi20
 export CAPILIB=$(TOPDIR)/../capi20
 export INSTALL=$(TOPDIR)/install-sh -c
-export INSTALLDATA=$(TOPDIR)/install-sh -c -o root -m 600
+export INSTALLDATA=$(TOPDIR)/install-sh -c 
 export MKDIR=$(TOPDIR)/mkinstalldirs
 
 PPPSRCDIRS=/src/isdn/pppd
diff -Naur isdn4k-utils.orig/pppdcapiplugin/Rules.make isdn4k-utils/pppdcapiplugin/Rules.make
--- isdn4k-utils.orig/pppdcapiplugin/Rules.make	Tue May  1 12:43:49 2001
+++ isdn4k-utils/pppdcapiplugin/Rules.make	Mon Oct  7 12:18:59 2002
@@ -13,18 +13,29 @@
 CC	= gcc
 INC     = -I$(TOPDIR) -I$(CAPIINC) -Ipppd
 DEFS    = -DPPPVER=$(shell $(TOPDIR)/pversion $(PPPVERSION))
-CFLAGS	= -O2 -Wall -fPIC $(DEFS) $(INC) -L$(CAPILIB)
+CFLAGS	= -O2
 LDFLAGS	= -shared -L$(CAPILIB)
 
 ALL = capiplugin.so userpass.so
 
 all:	$(ALL)
 
+capiplugin.o: ../capiplugin.c
+	$(CC) $(CFLAGS) -Wall -fPIC $(DEFS) $(INC) -L$(CAPILIB) -o capiplugin.o -c ../capiplugin.c
+
+capiconn.o: ../capiconn.c
+	$(CC) $(CFLAGS) -Wall -fPIC $(DEFS) $(INC) -L$(CAPILIB) -o capiconn.o -c ../capiconn.c
+	
+
 capiplugin.so: capiplugin.o capiconn.o
 	$(CC) -o $@ $(LDFLAGS) capiplugin.o capiconn.o -lcapi20dyn
 
+userpass.o: ../userpass.c
+	$(CC) $(CFLAGS) -Wall -fPIC $(DEFS) $(INC) -L$(CAPILIB) -o userpass.o -c ../userpass.c
+	
+
 userpass.so: userpass.o
-	$(CC) -o $@ $(LDFLAGS) $(CFLAGS) -nostdlib userpass.o
+	$(CC) -o $@ $(LDFLAGS) $(CFLAGS) -Wall -fPIC $(DEFS) $(INC) -L$(CAPILIB) -nostdlib userpass.o
 
 distclean: clean
 
