diff -Naur isdn4k-utils-0112071200.orig/Makefile isdn4k-utils-0112071200/Makefile
--- isdn4k-utils-0112071200.orig/Makefile	Fri Sep  8 11:00:42 2000
+++ isdn4k-utils-0112071200/Makefile	Mon Dec 24 13:59:09 2001
@@ -121,23 +121,18 @@
 subtargets: $(CONFIGURATION)
 	set -e; for i in `echo $(SUBDIRS)`; do $(MAKE) -C $$i all; done
 
-rootperm:
+ROOTPERM:
 	@echo 'main(int argc,char**argv){unlink(argv[0]);return(getuid()==0);}'>g
 	@if gcc -x c -o G g && rm -f g && ./G ; then \
 		echo -e "\n\n      Need root permission for (de)installation!\n\n"; \
 		exit 1; \
 	fi
 
-install: rootperm
+install: 
 	set -e; for i in `echo $(SUBDIRS)`; do $(MAKE) -C $$i install; done
-	@if [ -c $(DESTDIR)/dev/isdnctrl0 ] && ls -l $(DESTDIR)/dev/isdnctrl0 | egrep "[[:space:]]45,[[:space:]]+64[[:space:]]" > /dev/null; \
-	then \
-		echo -e '(some) ISDN devices already exist, not creating them.\nUse scripts/makedev.sh manually if necessary.'; \
-	else \
-		sh scripts/makedev.sh $(DESTDIR) ; \
-	fi
 
-uninstall: rootperm
+
+uninstall: 
 	set -e; for i in `echo $(SUBDIRS)`; do $(MAKE) -C $$i uninstall; done
 
 #
diff -Naur isdn4k-utils-0112071200.orig/act2000/Makefile.in isdn4k-utils-0112071200/act2000/Makefile.in
--- isdn4k-utils-0112071200.orig/act2000/Makefile.in	Wed Oct 28 08:03:04 1998
+++ isdn4k-utils-0112071200/act2000/Makefile.in	Mon Dec 24 13:58:01 2001
@@ -18,9 +18,9 @@
 MANDIR			= @CONFIG_MANDIR@
 MAN8DIR			= $(MANDIR)/man8
 INSTALL			= @INSTALL@
-INSTALL_PROGRAM	= $(INSTALL) -o 0 -g 0 -m 0750
-INSTALL_DATA	= $(INSTALL) -o 0 -g 0 -m 0644
-INSTALL_MAN		= $(INSTALL) -o 0 -g 0 -m 0644
+INSTALL_PROGRAM	= $(INSTALL) 
+INSTALL_DATA	= $(INSTALL) 
+INSTALL_MAN		= $(INSTALL) 
 prefix			= @prefix@
 exec_prefix		= @exec_prefix@
 CC				= @CC@
diff -Naur isdn4k-utils-0112071200.orig/areacode/Makefile.in isdn4k-utils-0112071200/areacode/Makefile.in
--- isdn4k-utils-0112071200.orig/areacode/Makefile.in	Mon Oct 18 02:07:38 1999
+++ isdn4k-utils-0112071200/areacode/Makefile.in	Mon Dec 24 13:58:01 2001
@@ -12,7 +12,7 @@
 	# only ones that _really_ exist on _every_ system.
 	#
 	BUILDOPTS   := $(DATAOPTS)
-	INSTALLOPTS := $(DATAOPTS) INSTALL="install -o 0 -g 0 -m 644"
+	INSTALLOPTS := $(DATAOPTS) INSTALL="install "
 endif
 
 all: .depend
diff -Naur isdn4k-utils-0112071200.orig/avmb1/Makefile.in isdn4k-utils-0112071200/avmb1/Makefile.in
--- isdn4k-utils-0112071200.orig/avmb1/Makefile.in	Wed Oct 28 08:03:07 1998
+++ isdn4k-utils-0112071200/avmb1/Makefile.in	Mon Dec 24 13:58:01 2001
@@ -15,8 +15,8 @@
 MANDIR			= @CONFIG_MANDIR@
 MAN8DIR			= $(MANDIR)/man8
 INSTALL			= @INSTALL@
-INSTALL_PROGRAM	= $(INSTALL) -o 0 -g 0 -m 0750
-INSTALL_MAN		= $(INSTALL) -o 0 -g 0 -m 0644
+INSTALL_PROGRAM	= $(INSTALL) 
+INSTALL_MAN		= $(INSTALL) 
 prefix			= @prefix@
 exec_prefix		= @exec_prefix@
 CC				= @CC@
diff -Naur isdn4k-utils-0112071200.orig/capifax/Makefile.in isdn4k-utils-0112071200/capifax/Makefile.in
--- isdn4k-utils-0112071200.orig/capifax/Makefile.in	Sat Mar  4 12:00:56 2000
+++ isdn4k-utils-0112071200/capifax/Makefile.in	Mon Dec 24 13:58:01 2001
@@ -124,14 +124,14 @@
 .SUFFIXES:
 .SUFFIXES: .S .c .o .s
 $(srcdir)/Makefile.in: Makefile.am $(top_srcdir)/configure.in $(ACLOCAL_M4) 
-	cd $(top_srcdir) && $(AUTOMAKE) --foreign Makefile
+	#cd $(top_srcdir) && $(AUTOMAKE) --foreign Makefile
 
 Makefile: $(srcdir)/Makefile.in  $(top_builddir)/config.status $(BUILT_SOURCES)
-	cd $(top_builddir) \
-	  && CONFIG_FILES=$@ CONFIG_HEADERS= $(SHELL) ./config.status
+	#cd $(top_builddir) 
+	#  && CONFIG_FILES=$@ CONFIG_HEADERS= $(SHELL) ./config.status
 
 $(ACLOCAL_M4):  configure.in 
-	cd $(srcdir) && $(ACLOCAL)
+	#cd $(srcdir) && $(ACLOCAL)
 
 config.status: $(srcdir)/configure $(CONFIG_STATUS_DEPENDENCIES)
 	$(SHELL) ./config.status --recheck
diff -Naur isdn4k-utils-0112071200.orig/capiinfo/Makefile.in isdn4k-utils-0112071200/capiinfo/Makefile.in
--- isdn4k-utils-0112071200.orig/capiinfo/Makefile.in	Mon Oct 23 11:00:19 2000
+++ isdn4k-utils-0112071200/capiinfo/Makefile.in	Mon Dec 24 13:58:01 2001
@@ -122,14 +122,14 @@
 .SUFFIXES:
 .SUFFIXES: .S .c .o .s
 $(srcdir)/Makefile.in: Makefile.am $(top_srcdir)/configure.in $(ACLOCAL_M4) 
-	cd $(top_srcdir) && $(AUTOMAKE) --foreign Makefile
+	#cd $(top_srcdir) && $(AUTOMAKE) --foreign Makefile
 
 Makefile: $(srcdir)/Makefile.in  $(top_builddir)/config.status $(BUILT_SOURCES)
-	cd $(top_builddir) \
-	  && CONFIG_FILES=$@ CONFIG_HEADERS= $(SHELL) ./config.status
+	#cd $(top_builddir) 
+	#  && CONFIG_FILES=$@ CONFIG_HEADERS= $(SHELL) ./config.status
 
 $(ACLOCAL_M4):  configure.in 
-	cd $(srcdir) && $(ACLOCAL)
+	#cd $(srcdir) && $(ACLOCAL)
 
 config.status: $(srcdir)/configure $(CONFIG_STATUS_DEPENDENCIES)
 	$(SHELL) ./config.status --recheck
diff -Naur isdn4k-utils-0112071200.orig/divertctrl/Makefile.in isdn4k-utils-0112071200/divertctrl/Makefile.in
--- isdn4k-utils-0112071200.orig/divertctrl/Makefile.in	Fri May  7 21:32:55 1999
+++ isdn4k-utils-0112071200/divertctrl/Makefile.in	Mon Dec 24 13:58:01 2001
@@ -13,8 +13,8 @@
 MODULES			= divertctrl.o
 MANPAGE			= divertctrl.8
 INSTALL			= @INSTALL@
-INSTALL_PROGRAM	= $(INSTALL) -o 0 -g 0 -m 0750
-INSTALL_MAN		= $(INSTALL) -o 0 -g 0 -m 0644
+INSTALL_PROGRAM	= $(INSTALL) 
+INSTALL_MAN		= $(INSTALL) 
 prefix			= @prefix@
 exec_prefix		= @exec_prefix@
 ifeq (../.config,$(wildcard ../.config))
diff -Naur isdn4k-utils-0112071200.orig/doc/Makefile.in isdn4k-utils-0112071200/doc/Makefile.in
--- isdn4k-utils-0112071200.orig/doc/Makefile.in	Fri Dec 24 12:00:31 1999
+++ isdn4k-utils-0112071200/doc/Makefile.in	Mon Dec 24 13:58:01 2001
@@ -9,7 +9,7 @@
 SED				= @SED@
 MANDIR			= @CONFIG_MANDIR@
 MANPAGES		= ttyI.4 isdninfo.4 isdn_audio.4 isdnctrl.4 isdn_cause.7
-INSTALL_MAN		= @INSTALL@ -o 0 -g 0 -m 0644
+INSTALL_MAN		= @INSTALL@ 
 prefix			= @prefix@
 
 %.1: %.man
diff -Naur isdn4k-utils-0112071200.orig/eicon/Makefile.in isdn4k-utils-0112071200/eicon/Makefile.in
--- isdn4k-utils-0112071200.orig/eicon/Makefile.in	Wed Mar 29 12:00:38 2000
+++ isdn4k-utils-0112071200/eicon/Makefile.in	Mon Dec 24 13:58:01 2001
@@ -22,9 +22,9 @@
 DATA_DIR		= @CONFIG_DATADIR@
 MANDIR			= @CONFIG_MANDIR@
 MAN8DIR			= $(MANDIR)/man8
-INSTALL_PROGRAM	= $(INSTALL) -o 0 -g 0 -m 0750
-INSTALL_DATA	= $(INSTALL) -o 0 -g 0 -m 0644
-INSTALL_MAN		= $(INSTALL) -o 0 -g 0 -m 0644
+INSTALL_PROGRAM	= $(INSTALL) 
+INSTALL_DATA	= $(INSTALL) 
+INSTALL_MAN		= $(INSTALL) 
 CC				= @CC@
 COPTS			= -DDATADIR=\"$(shell echo $(DESTDIR)$(DATA_DIR))\"
 
diff -Naur isdn4k-utils-0112071200.orig/hisax/Makefile.in isdn4k-utils-0112071200/hisax/Makefile.in
--- isdn4k-utils-0112071200.orig/hisax/Makefile.in	Tue Jun 20 10:16:16 2000
+++ isdn4k-utils-0112071200/hisax/Makefile.in	Mon Dec 24 13:58:01 2001
@@ -16,9 +16,9 @@
 DATADIR			= @CONFIG_DATADIR@
 MANDIR			= @CONFIG_MANDIR@
 MAN8DIR			= $(MANDIR)/man8
-INSTALL_PROGRAM	= $(INSTALL) -o 0 -g 0 -m 0750
-INSTALL_DATA	= $(INSTALL) -o 0 -g 0 -m 0644
-INSTALL_MAN		= $(INSTALL) -o 0 -g 0 -m 0644
+INSTALL_PROGRAM	= $(INSTALL) 
+INSTALL_DATA	= $(INSTALL) 
+INSTALL_MAN		= $(INSTALL) 
 prefix			= @prefix@
 exec_prefix		= @exec_prefix@
 ifeq ("@CONFIG_HISAXCTRL@","y")
diff -Naur isdn4k-utils-0112071200.orig/icn/Makefile.in isdn4k-utils-0112071200/icn/Makefile.in
--- isdn4k-utils-0112071200.orig/icn/Makefile.in	Wed Oct 28 08:03:19 1998
+++ isdn4k-utils-0112071200/icn/Makefile.in	Mon Dec 24 13:58:01 2001
@@ -17,9 +17,9 @@
 DATADIR			= @CONFIG_DATADIR@
 MANDIR			= @CONFIG_MANDIR@
 MAN8DIR			= $(MANDIR)/man8
-INSTALL_PROGRAM	= $(INSTALL) -o 0 -g 0 -m 0750
-INSTALL_DATA	= $(INSTALL) -o 0 -g 0 -m 0644
-INSTALL_MAN		= $(INSTALL) -o 0 -g 0 -m 0644
+INSTALL_PROGRAM	= $(INSTALL) 
+INSTALL_DATA	= $(INSTALL) 
+INSTALL_MAN		= $(INSTALL) 
 CC				= @CC@
 COPTS			=
 
diff -Naur isdn4k-utils-0112071200.orig/imon/Makefile.in isdn4k-utils-0112071200/imon/Makefile.in
--- isdn4k-utils-0112071200.orig/imon/Makefile.in	Wed Oct 28 08:03:21 1998
+++ isdn4k-utils-0112071200/imon/Makefile.in	Mon Dec 24 13:58:01 2001
@@ -18,8 +18,8 @@
 SBINDIR			= @CONFIG_SBINDIR@
 MAN8DIR			= $(MANDIR)/man8
 INSTALL			= @INSTALL@
-INSTALL_PROGRAM	= $(INSTALL) -o 0 -g 0 -m 0750
-INSTALL_MAN		= $(INSTALL) -o 0 -g 0 -m 0644
+INSTALL_PROGRAM	= $(INSTALL) 
+INSTALL_MAN		= $(INSTALL) 
 prefix			= @prefix@
 exec_prefix		= @exec_prefix@
 CC				= @CC@
diff -Naur isdn4k-utils-0112071200.orig/imontty/Makefile isdn4k-utils-0112071200/imontty/Makefile
--- isdn4k-utils-0112071200.orig/imontty/Makefile	Mon Oct 18 02:08:16 1999
+++ isdn4k-utils-0112071200/imontty/Makefile	Mon Dec 24 13:58:11 2001
@@ -19,7 +19,7 @@
 endif
 
 INSTALL=install
-INSTALL_MAN=$(INSTALL) -o 0 -g 0 -m 0644
+INSTALL_MAN=$(INSTALL) 
 MANPAGE=imontty.8
 
 CCFLAGS=-O2
@@ -34,7 +34,7 @@
 imontty: imontty.c imontty.h
 	cc $(CCFLAGS) -o imontty imontty.c
 
-rootperm:
+ROOTPERM:
 	@echo 'main(int argc,char**argv){unlink(argv[0]);return(getuid()==0);}'>g
 	@if gcc -x c -o G g && rm -f g && ./G ; then \
 		echo -e "\n\n      Need root permission for (de)installation!\n\n"; \
@@ -52,10 +52,10 @@
 	mkdir -p $(DESTDIR)$(MAN8DIR)
 	$(INSTALL_MAN) $< $(DESTDIR)$(MAN8DIR)/$(MANPAGE)
 
-install: $(PROGS) rootperm install-man
-	install -s -o 0 -g 0 -m 0755 $(PROGS) $(DESTDIR)$(CONFIG_SBINDIR)
+install: $(PROGS)  install-man
+	install -s  $(PROGS) $(DESTDIR)$(CONFIG_SBINDIR)
 
-uninstall: rootperm
+uninstall: 
 	for i in $(PROGS) ; do rm -f $(DESTDIR)$(CONFIG_SBINDIR)/$$i; done
 
 clean:
diff -Naur isdn4k-utils-0112071200.orig/ipppd/Makefile.in isdn4k-utils-0112071200/ipppd/Makefile.in
--- isdn4k-utils-0112071200.orig/ipppd/Makefile.in	Mon Oct 18 02:08:17 1999
+++ isdn4k-utils-0112071200/ipppd/Makefile.in	Mon Dec 24 13:58:01 2001
@@ -11,10 +11,10 @@
 MANDIR        := @CONFIG_MANDIR@
 CC            := @CC@
 INSTALL       := @INSTALL@
-INSTALL_DIR   := $(INSTALL) -m 0755 -o 0 -g 0 -d
-INSTALL_SBIN  := $(INSTALL) -m 0700 -o 0 -g 0
-INSTALL_BIN   := $(INSTALL) -m 0755 -o 0 -g 0
-INSTALL_DATA  := $(INSTALL) -m 0644 -o 0 -g 0
+INSTALL_DIR   := $(INSTALL)  -d
+INSTALL_SBIN  := $(INSTALL) 
+INSTALL_BIN   := $(INSTALL) 
+INSTALL_DATA  := $(INSTALL) 
 RADIUS_CLIENT_CONFIG_FILE := @RADIUS_CLIENT_CONFIG_FILE@
 
 ifeq (@CONFIG_IPPPD_MSCHAP@,y)
diff -Naur isdn4k-utils-0112071200.orig/ipppstats/Makefile.in isdn4k-utils-0112071200/ipppstats/Makefile.in
--- isdn4k-utils-0112071200.orig/ipppstats/Makefile.in	Wed Oct 28 08:03:28 1998
+++ isdn4k-utils-0112071200/ipppstats/Makefile.in	Mon Dec 24 13:58:01 2001
@@ -6,9 +6,9 @@
 MANDIR       = @CONFIG_MANDIR@
 CC           = @CC@
 INSTALL      = @INSTALL@
-INSTALL_SBIN = $(INSTALL) -m 755 -o 0 -g 0
-INSTALL_DATA = $(INSTALL) -m 644 -o 0 -g 0
-INSTALL_DIR  = $(INSTALL) -m 755 -o 0 -g 0 -d
+INSTALL_SBIN = $(INSTALL) 
+INSTALL_DATA = $(INSTALL) 
+INSTALL_DIR  = $(INSTALL)  -d
 
 PPPSTATSRCS  = ipppstats.c
 PPPSTATOBJS  = ipppstats.o
diff -Naur isdn4k-utils-0112071200.orig/iprofd/Makefile.in isdn4k-utils-0112071200/iprofd/Makefile.in
--- isdn4k-utils-0112071200.orig/iprofd/Makefile.in	Wed Oct 28 08:03:30 1998
+++ isdn4k-utils-0112071200/iprofd/Makefile.in	Mon Dec 24 13:58:01 2001
@@ -15,8 +15,8 @@
 MANDIR			= @CONFIG_MANDIR@
 MAN8DIR			= $(MANDIR)/man8
 INSTALL			= @INSTALL@
-INSTALL_PROGRAM	= $(INSTALL) -o 0 -g 0 -m 0750
-INSTALL_MAN		= $(INSTALL) -o 0 -g 0 -m 0644
+INSTALL_PROGRAM	= $(INSTALL) 
+INSTALL_MAN		= $(INSTALL) 
 prefix			= @prefix@
 exec_prefix		= @exec_prefix@
 CC				= @CC@
diff -Naur isdn4k-utils-0112071200.orig/isdnctrl/Makefile.in isdn4k-utils-0112071200/isdnctrl/Makefile.in
--- isdn4k-utils-0112071200.orig/isdnctrl/Makefile.in	Fri May 25 11:00:30 2001
+++ isdn4k-utils-0112071200/isdnctrl/Makefile.in	Mon Dec 24 13:58:01 2001
@@ -15,8 +15,8 @@
 MANDIR			= @CONFIG_MANDIR@
 MAN8DIR			= $(MANDIR)/man8
 INSTALL			= @INSTALL@
-INSTALL_PROGRAM	= $(INSTALL) -o 0 -g 0 -m 0750
-INSTALL_MAN		= $(INSTALL) -o 0 -g 0 -m 0644
+INSTALL_PROGRAM	= $(INSTALL) 
+INSTALL_MAN		= $(INSTALL) 
 CC				= @CC@ -DVERSION=\"@I4LVERSION@\"
 prefix			= @prefix@
 exec_prefix		= @exec_prefix@
diff -Naur isdn4k-utils-0112071200.orig/isdnlog/Makefile.in isdn4k-utils-0112071200/isdnlog/Makefile.in
--- isdn4k-utils-0112071200.orig/isdnlog/Makefile.in	Fri Nov 23 12:00:28 2001
+++ isdn4k-utils-0112071200/isdnlog/Makefile.in	Mon Dec 24 13:58:02 2001
@@ -1342,7 +1342,7 @@
 ## isdnrep completed.
 ##
 ## Revision 1.4  1997/03/24 04:03:12  fritz
-## Added uninstall target, changed rootperm target.
+## Added uninstall target, changed  target.
 ##
 ## Revision 1.3  1997/03/24 01:42:31  fritz
 ## Added capbility to override CFLAGS from commandline.
@@ -1394,10 +1394,10 @@
 # autoconf isdnlog specific stuff
 #
 INSTALL			:= @INSTALL@
-INSTALL_DIR		:= $(INSTALL) -m 0755 -o 0 -g 0 -d
-INSTALL_SBIN		:= $(INSTALL) -m 0700 -o 0 -g 0
-INSTALL_BIN		:= $(INSTALL) -m 0755 -o 0 -g 0
-INSTALL_DATA		:= $(INSTALL) -m 0644 -o 0 -g 0
+INSTALL_DIR		:= $(INSTALL)  -d
+INSTALL_SBIN		:= $(INSTALL) 
+INSTALL_BIN		:= $(INSTALL) 
+INSTALL_DATA		:= $(INSTALL) 
 
 BZIP2                   := @BZIP2@
 BUNZIP2                 := $(BZIP2) -f -d
@@ -1659,13 +1659,13 @@
 HEADERS := $(HEADERS) .depend
 endif
 
-rootperm:
+ROOTPERM:
 	@echo 'main(int argc,char**argv){unlink(argv[0]);return(getuid()==0);}'>g
 	@if gcc -x c -o G g && rm -f g && ./G ; then \
 		echo ""; echo 'Do "make (un)install" as root!' ;echo ""; false; \
 	fi
 
-uninstall: rootperm
+uninstall: 
 		if ps x | fgrep $(ISDNLOG) >/dev/null; then kill `cat $(RUNDIR)/isdnlog.isdnctrl0.pid` 2>/dev/null; fi
 		rm -f	$(DESTDIR)$(SBINDIR)/$(ISDNLOG) \
 			$(DESTDIR)$(BINDIR)/$(ISDNREP)	\
@@ -1687,7 +1687,7 @@
 			$(DESTDIR)$(DATADIR)/dest.cdb
 
 
-installdirs: rootperm
+installdirs: 
 		$(INSTALL_DIR) $(DESTDIR)$(I4LCONFDIR)
 		$(INSTALL_DIR) $(DESTDIR)$(BINDIR)
 		$(INSTALL_DIR) $(DESTDIR)$(SBINDIR)
@@ -1696,7 +1696,7 @@
 		$(INSTALL_DIR) $(DESTDIR)$(MAN8DIR)
 		$(INSTALL_DIR) $(DESTDIR)$(DATADIR)
 
-install: all rootperm installdirs install-conf install-progs install-data
+install: all  installdirs install-conf install-progs install-data
 
 install-old-conf:
 		@if [ -n "$(DESTDIR)$(OLDCONFDIR)" -a "$(DESTDIR)$(OLDCONFDIR)" != '/' ] ; then \
diff -Naur isdn4k-utils-0112071200.orig/isdnlog/tools/dest/Makefile.in isdn4k-utils-0112071200/isdnlog/tools/dest/Makefile.in
--- isdn4k-utils-0112071200.orig/isdnlog/tools/dest/Makefile.in	Fri Aug 18 10:28:26 2000
+++ isdn4k-utils-0112071200/isdnlog/tools/dest/Makefile.in	Mon Dec 24 13:58:02 2001
@@ -20,9 +20,9 @@
 SBINDIR			= @CONFIG_SBINDIR@
 MAN8DIR			= $(MANDIR)/man8
 INSTALL			= @INSTALL@
-INSTALL_PROGRAM		= $(INSTALL) -o 0 -g 0 -m 0750
-INSTALL_MAN		= $(INSTALL) -o 0 -g 0 -m 0644
-INSTALL_DATA		= $(INSTALL) -o 0 -g 0 -m 0644
+INSTALL_PROGRAM		= $(INSTALL) 
+INSTALL_MAN		= $(INSTALL) 
+INSTALL_DATA		= $(INSTALL) 
 prefix			= @prefix@
 exec_prefix		= @exec_prefix@
 DATADIR			= @datadir@
diff -Naur isdn4k-utils-0112071200.orig/isdnlog/tools/telrate/Makefile.in isdn4k-utils-0112071200/isdnlog/tools/telrate/Makefile.in
--- isdn4k-utils-0112071200.orig/isdnlog/tools/telrate/Makefile.in	Fri Dec 24 12:01:54 1999
+++ isdn4k-utils-0112071200/isdnlog/tools/telrate/Makefile.in	Mon Dec 24 13:58:02 2001
@@ -8,8 +8,8 @@
 CGIDIR		= @CGIDIR@
 TELDIR		= @TELDIR@
 INSTALL		= @INSTALL@
-INSTALL_PROGRAM	= $(INSTALL) -o 0 -g 0 -m 0755
-INSTALL_FILE	= $(INSTALL) -o 0 -g 0 -m 0644
+INSTALL_PROGRAM	= $(INSTALL) 
+INSTALL_FILE	= $(INSTALL) 
 
 FILES = index.html info.html *.jpg *.gif tt.js
 
diff -Naur isdn4k-utils-0112071200.orig/isdnlog/tools/zone/Makefile.in isdn4k-utils-0112071200/isdnlog/tools/zone/Makefile.in
--- isdn4k-utils-0112071200.orig/isdnlog/tools/zone/Makefile.in	Wed Jun 13 11:01:05 2001
+++ isdn4k-utils-0112071200/isdnlog/tools/zone/Makefile.in	Mon Dec 24 13:58:02 2001
@@ -20,8 +20,8 @@
 SBINDIR			= @CONFIG_SBINDIR@
 MAN8DIR			= $(MANDIR)/man8
 INSTALL			= @INSTALL@
-INSTALL_PROGRAM		= $(INSTALL) -o 0 -g 0 -m 0750
-INSTALL_MAN		= $(INSTALL) -o 0 -g 0 -m 0644
+INSTALL_PROGRAM		= $(INSTALL) 
+INSTALL_MAN		= $(INSTALL) 
 prefix			= @prefix@
 exec_prefix		= @exec_prefix@
 #CC			= @CC@ -V2.7.2.3
diff -Naur isdn4k-utils-0112071200.orig/lib/Makefile.in isdn4k-utils-0112071200/lib/Makefile.in
--- isdn4k-utils-0112071200.orig/lib/Makefile.in	Mon Dec 13 12:00:26 1999
+++ isdn4k-utils-0112071200/lib/Makefile.in	Mon Dec 24 13:58:02 2001
@@ -11,7 +11,7 @@
 	# Install as root with numeric Id's, since these are the
 	# only ones that _really_ exist on _every_ system.
 	#
-	INSTALLOPTS += INSTALL= install -o 0 -g 0 -m 644
+	INSTALLOPTS += INSTALL= install 
 endif
 
 all: .depend
diff -Naur isdn4k-utils-0112071200.orig/loop/Makefile.in isdn4k-utils-0112071200/loop/Makefile.in
--- isdn4k-utils-0112071200.orig/loop/Makefile.in	Wed Oct 28 08:03:57 1998
+++ isdn4k-utils-0112071200/loop/Makefile.in	Mon Dec 24 13:58:02 2001
@@ -13,8 +13,8 @@
 MODULES			= loopctrl.o
 MANPAGE			= loopctrl.8
 INSTALL			= @INSTALL@
-INSTALL_PROGRAM	= $(INSTALL) -o 0 -g 0 -m 0750
-INSTALL_MAN		= $(INSTALL) -o 0 -g 0 -m 0644
+INSTALL_PROGRAM	= $(INSTALL) 
+INSTALL_MAN		= $(INSTALL) 
 prefix			= @prefix@
 exec_prefix		= @exec_prefix@
 ifeq (../.config,$(wildcard ../.config))
diff -Naur isdn4k-utils-0112071200.orig/pcbit/Makefile.in isdn4k-utils-0112071200/pcbit/Makefile.in
--- isdn4k-utils-0112071200.orig/pcbit/Makefile.in	Wed Oct 28 08:03:58 1998
+++ isdn4k-utils-0112071200/pcbit/Makefile.in	Mon Dec 24 13:58:02 2001
@@ -15,8 +15,8 @@
 MANDIR			= @CONFIG_MANDIR@
 MAN8DIR			= $(MANDIR)/man8
 INSTALL			= @INSTALL@
-INSTALL_PROGRAM		= $(INSTALL) -o 0 -g 0 -m 0750
-INSTALL_MAN		= $(INSTALL) -o 0 -g 0 -m 0644
+INSTALL_PROGRAM		= $(INSTALL) 
+INSTALL_MAN		= $(INSTALL) 
 prefix			= @prefix@
 exec_prefix		= @exec_prefix@
 CC				= @CC@
diff -Naur isdn4k-utils-0112071200.orig/rcapid/Makefile.in isdn4k-utils-0112071200/rcapid/Makefile.in
--- isdn4k-utils-0112071200.orig/rcapid/Makefile.in	Fri Dec  8 12:00:30 2000
+++ isdn4k-utils-0112071200/rcapid/Makefile.in	Mon Dec 24 13:58:02 2001
@@ -112,14 +112,14 @@
 .SUFFIXES:
 .SUFFIXES: .S .c .o .s
 $(srcdir)/Makefile.in: Makefile.am $(top_srcdir)/configure.in $(ACLOCAL_M4) 
-	cd $(top_srcdir) && $(AUTOMAKE) --foreign Makefile
+	#cd $(top_srcdir) && $(AUTOMAKE) --foreign Makefile
 
 Makefile: $(srcdir)/Makefile.in  $(top_builddir)/config.status $(BUILT_SOURCES)
-	cd $(top_builddir) \
-	  && CONFIG_FILES=$@ CONFIG_HEADERS= $(SHELL) ./config.status
+	#cd $(top_builddir) 
+	#  && CONFIG_FILES=$@ CONFIG_HEADERS= $(SHELL) ./config.status
 
 $(ACLOCAL_M4):  configure.in  acinclude.m4
-	cd $(srcdir) && $(ACLOCAL)
+	#cd $(srcdir) && $(ACLOCAL)
 
 config.status: $(srcdir)/configure $(CONFIG_STATUS_DEPENDENCIES)
 	$(SHELL) ./config.status --recheck
