--- isdn4k-utils/pppdcapiplugin/Makefile.orig	Fri Apr  2 14:48:38 2004
+++ isdn4k-utils/pppdcapiplugin/Makefile	Fri Apr  2 15:11:58 2004
@@ -17,6 +17,6 @@
 MANDIR=/usr/share/man
 
 #PPPVERSION=$(shell /usr/sbin/pppd --version 2>&1 | sed -e "s/pppd\ version\ //")
-PLUGINDIR=$(DESTDIR)/usr/lib/pppd/$(PPPVERSION)
+PLUGINDIR=$(DESTDIR)$(CONFIG_LIBDIR)/pppd/$(PPPVERSION)
 include $(TOPDIR)/Rules.make
 
--- isdn4k-utils/Makefile.orig	Fri Apr  2 14:48:38 2004
+++ isdn4k-utils/Makefile	Fri Apr  2 15:11:32 2004
@@ -202,7 +202,7 @@
 	@set -e; for i in `echo $(BUILD_ONLY) $(SUBDIRS)`; do \
 		if [ -x $$i/configure ] ; then \
 			/bin/echo -e "\nRunning configure in $$i ...\n"; sleep 1; \
-			(cd $$i; ./configure --sbindir=$(CONFIG_SBINDIR) --bindir=$(CONFIG_BINDIR) --mandir=$(CONFIG_MANDIR) --datadir=$(CONFIG_DATADIR) || $(MAKE) -C ../ ERRDIR=$$i cfgerror); \
+			(cd $$i; ./configure --sbindir=$(CONFIG_SBINDIR) --bindir=$(CONFIG_BINDIR) --mandir=$(CONFIG_MANDIR) --datadir=$(CONFIG_DATADIR) --libdir=$(CONFIG_LIBDIR) || $(MAKE) -C ../ ERRDIR=$$i cfgerror); \
 		elif [ -f $$i/Makefile.in ] ; then \
 			/bin/echo -e "\nRunning make -f Makefile.in config in $$i ...\n"; sleep 1; \
 			$(MAKE) -C $$i -f Makefile.in config; \
