--- isdn4k-utils/pppdcapiplugin/Makefile.orig	2013-01-27 09:27:33.333885560 +0100
+++ isdn4k-utils/pppdcapiplugin/Makefile	2013-01-27 10:09:25.173833208 +0100
@@ -8,13 +8,15 @@
 #  as published by the Free Software Foundation; either version
 #  2 of the License, or (at your option) any later version.
 
-export TOPDIR=$(shell pwd)
-export CAPIINC=$(TOPDIR)/../capi20
-export CAPILIB=$(TOPDIR)/../capi20
-export INSTALL=$(TOPDIR)/install-sh -c
-export INSTALLDATA=$(TOPDIR)/install-sh -c -m 600
-export MKDIR=$(TOPDIR)/mkinstalldirs
-export PPPINC=/usr/include/pppd
+TOPDIR=$(shell pwd)
+CAPIINC=$(TOPDIR)/../capi20
+CAPILIB=$(TOPDIR)/../capi20
+INSTALL=$(TOPDIR)/install-sh -c
+INSTALLDATA=$(TOPDIR)/install-sh -c -m 600
+MKDIR=$(TOPDIR)/mkinstalldirs
+PPPINC=/usr/include/pppd
+MANDIR=/usr/share/man
+PLUGINDIR=$(DESTDIR)$(CONFIG_LIBDIR)/pppd/plugins
 
 ifeq ($(PPPVERSION),)
 PPPVERSION_XX=$(shell grep VERSION $(PPPINC)/patchlevel.h)
@@ -28,70 +30,9 @@
 
 ALL = capiplugin.so userpass.so
 
-#all: pversion
-#	@for i in $(PPPVERSIONS) ; do \
-#	    echo $(MAKE) -C ppp-$$i PPPVERSION=$$i $@ ; \
-#	    $(MAKE) -C ppp-$$i PPPVERSION=$$i $@ ; \
-#	done
-
-all: pversion ppp-$(PPPVERSION)
-	echo $(MAKE) -C ppp-$(PPPVERSION) PPPVERSION=$(PPPVERSION)
-	$(MAKE) -C ppp-$(PPPVERSION) PPPVERSION=$(PPPVERSION)
-
-ppp-$(PPPVERSION):
-	$(MKDIR) ppp-$(PPPVERSION)
-	echo created ppp-$(PPPVERSION)
-	cp Makefile.template ppp-$(PPPVERSION)/Makefile
+include $(TOPDIR)/Rules.make
 
 distclean: clean
 
-clean:
-	echo $(MAKE) -C ppp-$(PPPVERSION) PPPVERSION=$(PPPVERSION) $@ ;
-	$(MAKE) -C ppp-$(PPPVERSION) PPPVERSION=$(PPPVERSION) $@ ;
-
-install-peers:
-	$(MKDIR) $(PEERDIR)
-	@for i in $(PEERS); do \
-		echo $(INSTALLDATA) peers/$$i $(PEERDIR); \
-		$(INSTALLDATA) peers/$$i $(PEERDIR); \
-	done
-
-install-drdsl:
-	$(MKDIR) $(DRDSLDIR)
-	@for i in $(DRDSLFILES); do \
-		if [ -f $(DRDSLDIR)/$$i ] ; then \
-		   echo $(DRDSLDIR)/$$i already exists ; \
-		else \
-		  echo $(INSTALLDATA) examples/$$i $(DRDSLDIR); \
-		  $(INSTALLDATA) examples/$$i $(DRDSLDIR); \
-		fi \
-	done
-
-install-man:
-	for i in ${DESTDIR}/usr/share/man ${DESTDIR}/usr/man; do \
-		if [ -d $$i/man8 ] ; then \
-			echo $(INSTALLDATA) capiplugin.8 $$i/man8; \
-			$(INSTALLDATA) capiplugin.8 $$i/man8; \
-		fi ; \
-	done
-
-install:
-	echo $(MAKE) -C ppp-$(PPPVERSION) PPPVERSION=$(PPPVERSION) $@ ;
-	$(MAKE) -C ppp-$(PPPVERSION) PPPVERSION=$(PPPVERSION) $@ ;
-	$(MAKE) install-peers
-	$(MAKE) install-drdsl
-	$(MAKE) install-man
-
-srcsetup:
-	@for v in $(PPPVERSIONS) ; do \
-	    $(MKDIR) ppp-$$v/pppd ; \
-	    echo $(INSTALL) Makefile.template ppp-$$v/Makefile ; \
-	    $(INSTALL) Makefile.template ppp-$$v/Makefile ; \
-	    for h in pppd.h patchlevel.h ; do \
-	        echo $(INSTALL) $(PPPSRCDIRS)/ppp-$$v/pppd/$$h ppp-$$v/pppd ; \
-	        $(INSTALL) $(PPPSRCDIRS)/ppp-$$v/pppd/$$h ppp-$$v/pppd ; \
-	    done \
-	done
-
 config:
 	@echo nothing to configure
--- isdn4k-utils/pppdcapiplugin/Rules.make.orig	2013-01-27 08:06:26.000000000 +0100
+++ isdn4k-utils/pppdcapiplugin/Rules.make	2013-01-27 10:06:32.657170129 +0100
@@ -13,12 +13,21 @@
 CC	= gcc
 INC     = -I$(TOPDIR) -I$(CAPIINC) -I$(PPPINC)
 DEFS    = -DPPPVER=$(shell $(TOPDIR)/pversion $(PPPVERSION))
-MYCFLAGS= -O2 -Wall -fPIC $(DEFS) $(INC) -L$(CAPILIB)
+CCFLAGS = -O2
+CFLAGS  = $(CCFLAGS)
+MYCFLAGS= -Wall -fPIC $(DEFS) $(INC) -L$(CAPILIB)
 LDFLAGS	= -shared -L$(CAPILIB)
 
 ALL = capiplugin.so userpass.so
 
-all:	$(ALL)
+all:	def $(ALL)
+
+pversion: pversion.c
+	$(CC) $(CFLAGS) pversion.c -o pversion
+
+def: pversion
+	echo \#define PPPVER $(shell $(TOPDIR)/pversion $(PPPVERSION)) >> p.h
+	echo \#endif >> p.h
 
 capiplugin.so: capiplugin.o capiconn.o
 	$(CC) -o $@ $(LDFLAGS) capiplugin.o capiconn.o -lcapi20dyn
@@ -36,10 +45,12 @@
 
 install: $(ALL)
 	$(MKDIR) $(PLUGINDIR)
+	$(MKDIR) $(DESTDIR)/$(MANDIR)/man8
 	@for i in $(ALL); do \
 		echo $(INSTALL) $$i $(PLUGINDIR); \
 		$(INSTALL) $$i $(PLUGINDIR); \
 	done
+	$(INSTALL) capiplugin.8 $(DESTDIR)/$(MANDIR)/man8
 
 config:
 	@echo nothing to configure
diff -Naur isdn4k-utils.orig/pppdcapiplugin/capiconn.h isdn4k-utils/pppdcapiplugin/capiconn.h
--- isdn4k-utils.orig/pppdcapiplugin/capiconn.h	Wed Jun 11 23:06:50 2003
+++ isdn4k-utils/pppdcapiplugin/capiconn.h	Wed Jun 11 23:07:21 2003
@@ -25,7 +25,7 @@
 
 #ifndef __CAPICONN_H__
 #define __CAPICONN_H__
-
+#include "p.h"
 #include <stdarg.h>
 #include <capi20.h>
 
diff -Naur isdn4k-utils.orig/pppdcapiplugin/capiplugin.c isdn4k-utils/pppdcapiplugin/capiplugin.c
--- isdn4k-utils.orig/pppdcapiplugin/capiplugin.c	Wed Jun 11 23:06:50 2003
+++ isdn4k-utils/pppdcapiplugin/capiplugin.c	Wed Jun 11 23:07:21 2003
@@ -17,7 +17,7 @@
 #include <sys/ioctl.h>
 #include <sys/types.h>
 #include <sys/socket.h>
-#include "pppd.h"
+#include <pppd/pppd.h>
 #include "capiconn.h"
 #include <malloc.h>
 #include <string.h>
@@ -34,7 +34,7 @@
 
 /* -------------------------------------------------------------------- */
 
-#include "patchlevel.h"
+#include <pppd/patchlevel.h>
 #ifdef VERSION
 char pppd_version[] = VERSION;
 #endif
diff -Naur isdn4k-utils.orig/pppdcapiplugin/p.h isdn4k-utils/pppdcapiplugin/p.h
--- isdn4k-utils.orig/pppdcapiplugin/p.h	Thu Jan  1 01:00:00 1970
+++ isdn4k-utils/pppdcapiplugin/p.h	Wed Jun 11 23:07:21 2003
@@ -0,0 +1,2 @@
+#ifndef __P
+#define __P
diff -Naur isdn4k-utils.orig/pppdcapiplugin/userpass.c isdn4k-utils/pppdcapiplugin/userpass.c
--- isdn4k-utils.orig/pppdcapiplugin/userpass.c	Wed Jun 11 23:06:50 2003
+++ isdn4k-utils/pppdcapiplugin/userpass.c	Wed Jun 11 23:07:21 2003
@@ -30,9 +30,9 @@
  *
  *
  */
-#include "pppd.h"
+#include <pppd/pppd.h>
 
-#include "patchlevel.h"
+#include <pppd/patchlevel.h>
 #ifdef VERSION
 char pppd_version[] = VERSION;
 #endif
