diff -uNr isdn4k-utils.orig/capi20/capi20.c isdn4k-utils/capi20/capi20.c
--- isdn4k-utils.orig/capi20/capi20.c	Thu Mar  1 15:59:11 2001
+++ isdn4k-utils/capi20/capi20.c	Sun Aug 25 15:23:15 2002
@@ -367,7 +367,7 @@
 	          rcvbuf[0] = 30;
 	          rcvbuf[1] = 0;
 	       }
-	       data = (u_int64_t)rcvbuf + CAPIMSG_LEN(rcvbuf);
+	       data = rcvbuf + CAPIMSG_LEN(rcvbuf);
 	       rcvbuf[12] = rcvbuf[13] = rcvbuf[14] = rcvbuf[15] = 0;
 	       rcvbuf[22] = data & 0xff;
 	       rcvbuf[23] = (data >> 8) & 0xff;
